homeassistant/components/zha/manifest.json

```diff
@@ -95,6 +95,10 @@
     {
       "type": "_zigstar_gw._tcp.local.",
       "name": "*zigstar*"
+    },
+	{
```

Comment:
```suggestion
	  {
```

Reply:
Could there be a problem in the name slzb-06? That is, in the symbol `-`

Reply:
No, it's just failing the JSON linter. Make sure you set up `pre-commit`:

```console
$ pip install pre-commit
$ pre-commit install
```

If you modify `zha/manifest.json` again (i.e. add whitespace) and commit it, the hook should auto-format it correctly.

Reply:
Your JSON uses a tab character for indentation instead of spaces. Format it with `pre-commit`, it fixes this automatically.

Reply:
pre-commit throws an error in my case. I checked the source file, there are no tabs

Reply:
The linter doesn't lie 😄. It was originally indented with a single tab. When you committed my suggestion, it also added two spaces:

<img width="266" alt="image" src="https://user-images.githubusercontent.com/32534428/208182456-e6d0d33c-5092-4965-b5d8-91d2f8789669.png">

Reply:
Ok i got it. Looks like I checked the file after pre-commit formatted it XD